pkg/provisioner/ironic/testserver/ironic.go

```diff
@@ -58,3 +63,21 @@ func (m *IronicMock) WithDrivers() *IronicMock {
 	`)
 	return m
 }
+
+// WithNode configures the server with a valid response for /v1/nodes
+func (m *IronicMock) WithNode(node nodes.Node) *IronicMock {
```

Comment:
Some of these methods are already in master. Maybe this PR needs to be rebased?

Reply:
Done